assets/styles/style.css

```diff
@@ -0,0 +1,604 @@
+  background-color: #ebf0ee;
+}
+
+.thumbnail {
+  width: 100%;
+  height: 252px;
+  object-fit: cover;
+  object-position: center;
+}
+
+.portfolio-text,
+.card-text {
+  padding: 16px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+}
+
+.card-title {
+  font-size: 24px;
+  line-height: 32px;
+  font-weight: 600;
+}
+
+.categories {
+  list-style-type: none;
+  flex-wrap: wrap;
+  padding: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 8px;
+}
+
+.category {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  transition: 500ms ease;
+  font-size: 12px;
+  font-weight: 600;
+  padding: 12px;
+  text-decoration: none;
+  border-radius: 4px;
+  line-height: 16px;
+  height: 40px;
+}
+
+.about-me .category {
+  background-color: white;
+  color: #36b37f;
+}
+
+.portfolio-item .category {
+  background-color: #ebf0ee;
+  color: #172b4d;
+}
+
+.about-me .category:hover {
+  background-color: #36b37f;
+  color: white;
+}
+
+.portfolio-item .category:hover {
+  background-color: #36b37f;
+  color: white;
+}
+
+.button {
+  border-radius: 4px;
+  font-size: 17px;
+  line-height: 24px;
+  padding: 12px;
+  text-align: center;
+  text-decoration: none;
+  cursor: pointer;
+}
+
+.btn-primary {
+  background-color: #36b37f;
+  color: white;
+  transition: 500ms ease;
+}
+
+.btn-primary:hover {
+  background-color: #36b37f;
+}
+
+.btn-primary:active {
+  background-color: #008552;
+}
+
+.about-me {
+  margin-top: 0;
+  background-image: url("../images/about-me.png");
+  background-position: -49px -37px;
+  object-fit: cover;
+  padding: 64px 24px;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.about-text {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.about-icons {
+  height: auto;
+  width: 100%;
+}
+
+/* contact-form */
+.contact-me {
+  padding: 0 24px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-image: url("../images/illustration-contact-formfooter-mobile.png");
+  background-position: right -50px bottom 0;
+  background-repeat: no-repeat;
+}
+
+.my-contact-form {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.my-contact-form input,
+textarea {
+  width: 100%;
+  padding: 15px 16px;
+  margin-bottom: 24px;
+  border: 2px solid #d0d9d4 !important;
+  border-radius: 4px;
+}
+
+.my-contact-form input[type="text"]:focus,
+input[type="email"]:focus,
+textarea:focus {
+  border: 2px solid #36b37f;
+  outline: none;
+}
+
+input:invalid,
+textarea:invalid {
+  border: 2px solid red;
+}
+
+.mobile-form {
+  width: 100%;
+  height: auto;
+}
+
+@media screen and (max-width: 640px) {
+  .desktop-nav {
+    display: none;
+  }
+
+  .social-links-footer {
+    display: none;
+  }
+
+  hr {
+    display: none;
+  }
+}
+
+@media screen and (min-width: 768px) {
+  .wrapper {
+    max-width: 768px;
+  }
```

Comment:
You have done great adjusting the breakpoints of the mobile menu as requested by the previous reviewer. However, there still seems to be an overlap for screen sizes greater than 768px and less than 1024px. Kindly look into that too.


![image](https://user-images.githubusercontent.com/20554583/199678518-cbb987ec-c072-40f8-a276-25cd61ffe7b8.png)